cmd/minikube/cmd/root.go

```diff
@@ -78,8 +77,16 @@ var RootCmd = &cobra.Command{
 // Execute adds all child commands to the root command sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	defer audit.Log(time.Now())
-
+	auditID, err := audit.LogCommandStart()
+	if err != nil{
+		klog.Errorf("%v", err)
```

Comment:
```suggestion
		klog.Errorf("failed to log command start to audit: %v", err)
```
The error messages should be more descriptive for debugging purposes later.